src/main/java/org/apache/flink/state/benchmark/StateBenchmarkBase.java

```diff
@@ -57,7 +56,7 @@ public class StateBenchmarkBase extends BenchmarkBase {
     static AtomicInteger keyIndex;
     final ThreadLocalRandom random = ThreadLocalRandom.current();
 
-    @Param({"HEAP", "ROCKSDB"})
+    @Param({"HEAP", "ROCKSDB", "HEAP_CHANGELOG", "ROCKSDB_CHANGELOG"})
```

Comment:
I would remove `HEAP_CHANGELOG` from the runs. This sounds like a quite unlikely production combination.

Reply:
Thanks for the advice.
I aggree that we should focus on testing the production env.